### PR TITLE
Reworks immortality amulet void effect into a status effect

### PR DESCRIPTION
**Pull request by Chubbygummibear**
there was an exploit. this doesn't change how the spell or amulet work at all from the player's perspective

(i didn't get to do it and i'm sad)

OH AND I MADE IT MODULAR

# Changelog

:cl:  
rscadd: new status effect "Voided" (it's just the void magnet and immortality amulet void out effect)
bugfix: the icon is fixed for the genetics mutation spell
tweak: the voiding out starts and ends based on a status effect rather than a timer.
/:cl:

### CODE REVIEW

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -478,8 +483,9 @@ GLOBAL_LIST_EMPTY(bloodmen_list)
 
 /obj/effect/immortality_talisman/Initialize(mapload, mob/new_user)
 	. = ..()
-	if(new_user)
-		vanish(new_user)
+	// if(new_user && isliving(new_user))
+	// 	var/mob/living/L = new_user
```
Comment (ToasterBiome) [inline code/modules/mining/lavaland/necropolis_chests.dm]: what the dog doin

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -493,12 +499,12 @@ GLOBAL_LIST_EMPTY(bloodmen_list)
 
 	can_destroy = FALSE
 
-	addtimer(CALLBACK(src, .proc/unvanish, user), 10 SECONDS)
+	//addtimer(CALLBACK(src, .proc/unvanish, user), 10 SECONDS)
```
Comment (ToasterBiome) [inline code/modules/mining/lavaland/necropolis_chests.dm]: what the dog doin

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 1 above)
Comment (Chubbygummibear) [reply code/modules/mining/lavaland/necropolis_chests.dm]: fixed

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 2 above)
Comment (Chubbygummibear) [reply code/modules/mining/lavaland/necropolis_chests.dm]: shut

**Review by ToasterBiome — APPROVED** [review]
good

**Pull request merged**